### ISSUE TITLE
Update for Bootstrap v4

### ISSUE THREAD

**Issue by bmuenzenmeyer** [pattern-lab/starterkit-mustache-bootstrap#15]
https://getbootstrap.com/

**Comment by bmuenzenmeyer** [comment]
Some users expressed interest in this update - but it needs an active maintainer 😄

**Comment by bradfrost** [comment]
Yeah this would be interesting. I don't know what a port of Bootstrap 3 would entail, but I'm not willing to do the work myself. That said, I'd be happy to coach anyone who would want to take that project on.

**Comment by bmuenzenmeyer** [comment]
someone did in chat - prompting me to open this issue in the first place. have not heard from him or her since

**Comment by miklb** [comment]
I'm working with a few folks interested in using v4 of Bootstrap with Pattern Lab. We would be interested in contributing the work back if you have any pointers on where to start. I'm getting up to speed on the basics of the php versions now.

**Comment by bradfrost** [comment]
Hey thanks @miklb! Probably the easiest way to get started would be to create a new branch in this repo and update the styles and components to match Bootstrap v4. Lemme know if you hit any snags and I can hopefully get you unstuck.

**Comment by miklb** [comment]
@bradfrost I'm looking over https://github.com/pattern-lab/starterkit-mustache-demo and it seems there's been quite a bit of change to the mustache templates. How close should the mustache templates match or is that dependent on the bootstrap markup? Apologies if that's a dumb question, I'm still fully familiarizing myself with patternlab.

**Comment by bmuenzenmeyer** [comment]
The templates in thus repo can be whatever helps users get started with
bootstrap without being too opinionated. Regardless of the current demo. So
think atoms like buttons, badges.... molecules like cards. Anything
bootstrap. Sky is the limit!

**Comment by rachelreveley** [comment]
Did anything come of this? The place I work at is planning to incorporate pattern Lab into the workflow and we use BS 4.

**Comment by miklb** [comment]
Not on my end. Went another direction.

**Comment by bmuenzenmeyer** [comment]
> Did anything come of this? The place I work at is planning to incorporate pattern Lab into the workflow and we use BS 4.

sounds like a great first step would be giving this starterkit some attention, as it could be the basis of your future work. Mustache is not really the best of breed templating solution, however, so you may want to look into other templating engines, as constraints allow

**Comment by kentr** [comment]
I'm making a go at this, but with [twig](https://github.com/kentr/pattern-lab-starterkit-bootstrap4).
